src/visualization/types/Graph/view.tsx

```diff
@@ -181,13 +182,14 @@ const XYPlot: FC<Props> = ({
     return useVisYDomainSettings(storedDomain, memoizedYColumnData)
   }
 
+  const zoomQuery = useZoomQuery(properties)
```

Comment:
should this call be moved down so that it's inside the 

```ts
if (isFlagEnabled('zoomRequery') {
```
check?

Reply:
no, it's a hook and cannot be called conditionally.